:x:

```
======================================================================
FAIL: test_split_column (couchexport.tests.test_schema.SavedSchemaTest)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 93, in test_split_column
    self._test_split_column([[1, 1, 1, 1, None]])
  File "corehq/ex-submodules/couchexport/tests/test_schema.py", line 81, in _test_split_column
    self.assertEqual(data['Export']['headers'], expected_headers)
AssertionError: Lists differ: [] != ['Split | a', 'Split | b', 'Sp...
Second list contains 5 additional elements.
First extra element 0:
Split | a
- []
+ ['Split | a', 'Split | b', 'Split | c', 'Split | d', 'Split | extra']
----------------------------------------------------------------------
```

--- a/corehq/apps/reports/tests/test_form_export.py
+++ b/corehq/apps/reports/tests/test_form_export.py
@@ -51,7 +51,8 @@ class FormExportTest(TestCase):
             'tables': [{
                 'index': '#',
                 'display': 'Export',
-                'columns': [{'index': 'form.name', 'display': 'Name'}],
+                'columns': [{'index': 'form.name', 'display': 'Name',
+                             'show': True}],
             }]
         })
 

--- a/corehq/ex-submodules/couchexport/models.py
+++ b/corehq/ex-submodules/couchexport/models.py
@@ -378,7 +378,7 @@ class ExportTable(DocumentSchema):
         from couchexport.export import FormattedRow
         headers = []
         for col in self.columns:
-            if not col.show:
+            if (not col.show) and col.doc_type == 'ExportColumn':
                 continue  # ignore hidden columns
             if issubclass(type(col), ComplexExportColumn):
                 for header in col.get_headers():
@@ -436,7 +436,7 @@ class ExportTable(DocumentSchema):
             id = None
             cells = []
             for column, val in self.get_items_in_order(row):
-                if not column.show:
+                if (not column.show) and column.doc_type == 'ExportColumn':
                     continue  # don't add this column to the export
                 # TRANSFORM BABY!
                 if apply_transforms:

